examples/gno.land/p/demo/forms/forms.gno

```diff
@@ -0,0 +1,131 @@
+package forms
+
+import (
+	"errors"
+	"std"
+	"time"
+
+	"gno.land/p/demo/seqid"
+)
+
+type Field struct {
+	Label string
+
+	/*
```

Comment:
From what I can see, this is an example of how the fields would look like in JSON format. Try to extract this to godoc (maybe for this struct, maybe somewhere else), otherwise, it should be obvious from code that this is how it works

Reply:
https://github.com/gnolang/gno/pull/2604/commits/361170ae0e657adfa4c841faf839c7bbf0343cc9